pkgpanda/build/__init__.py

```diff
@@ -19,8 +19,8 @@
 from pkgpanda.constants import RESERVED_UNIT_NAMES
 from pkgpanda.exceptions import FetchError, PackageError, ValidationError
 from pkgpanda.util import (check_forbidden_services, download_atomic, load_json,
-                           load_string, make_file, make_tar, rewrite_symlinks,
-                           write_json, write_string)
+                           load_string, logger, make_file, make_tar,
```

Comment:
Ignore my previous comment. I meant an identation like this.

```
from pkgpanda.util import (
        check_forbidden_services,
        download_atomic,
        load_json,
        ...)
```

This was popularized by `go` and `gofmt` and has practical utility value. Some good libraries like `requests` use this style.  Then I realized our style checker supports this in an awkward way. So, let's just ignore my earlier comment.